app/policies/active_admin/page_policy.rb

```diff
@@ -0,0 +1,13 @@
+module ActiveAdmin
+  class PagePolicy < ApplicationPolicy
+    
+    def show?
+      case record.name
+      when 'Dashboard'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.